### PR TITLE
Paradigm generator for Spanish

### PR DESCRIPTION
**Pull request by albertonl**
Apart from the generator itself, I have added some stuff to the README and I have created an AUTHORS file to make the repo prettier.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1 @@
+2019, Alberto Navalon Lillo <albertonl.dev@gmail.com>
```
Comment (jonorthwash) [inline AUTHORS]: maybe also add the other two committers.

Comment (albertonl) [reply AUTHORS]: Yes, sure

**Comment by albertonl** [comment]
In the last two commits I have made all the changes you suggested me both via this PR and via GCI

**Comment by albertonl** [comment]
That last one is just a mess I did, and created another branch for the LICENSE file... Anyway, now it's good to go

**Comment by jonorthwash** [comment]
My (and Apertium's) go-to license is GPL.  Is there any reason you prefer MIT?

**Comment by albertonl** [comment]
No, not at all. I simply didn't know about the license. I'll change it to GPL 3

**Pull request merged**